### PR TITLE
799 add user agent

### PR DESCRIPTION
**Pull request by Orta21**
Ref: 799

### Description

PATCH: Add user agent for android webview

### Release Plan

- [ ] Released just merging into develop

### CODE REVIEW

File: packages/react-native-sdk/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/react-native-sdk",
-  "version": "1.0.4",
+  "version": "1.0.5",
```
Comment (leite08) [inline packages/react-native-sdk/package.json:3]: We were creating alpha versions before deploying in `prod`. NABD, just for alignment.

**Review by leite08 — COMMENTED** [review]
I don't have any context on why we're introducing this change. I assume its to fix an issue, but the issue is a generic one and there's no link to a thread on Slack or something where I can read more about the problem/goal.

**Comment by Orta21** [comment]
> I don't have any context on why we're introducing this change. I assume its to fix an issue, but the issue is a generic one and there's no link to a thread on Slack or something where I can read more about the problem/goal.

Gotcha the slack thread is in the Fold channel but to give more context the android app wasnt working because there was no User Agent so as a quick fix i added it and published a prod version to get this out the door

**Comment by leite08** [comment]
> Gotcha the slack thread is in the Fold channel but to give more context the android app wasnt working because there was no User Agent so as a quick fix i added it and published a prod version to get this out the door

Sounds good 👍  Next time I suggest just adding this small context and also stating the fact the code was already released in `prod`, its an important context for folks reviewing the PR. 😃

**Review by leite08 — APPROVED** [review]

**Pull request merged**